specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2019-01-21/kusto.json

```diff
@@ -0,0 +1,2226 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "KustoManagementClient",
+    "version": "2019-01-21"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Kusto/clusters/{clusterName}": {
+      "get": {
+        "tags": [
+          "Clusters"
+        ],
+        "operationId": "Clusters_Get",
+        "x-ms-examples": {
+          "KustoClustersGet": {
+            "$ref": "./examples/KustoClustersGet.json"
+          }
+        },
+        "description": "Gets a Kusto cluster.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Kusto cluster.",
+            "schema": {
+              "$ref": "#/definitions/Cluster"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Clusters"
+        ],
+        "operationId": "Clusters_CreateOrUpdate",
+        "x-ms-examples": {
+          "KustoClustersCreateOrUpdate": {
+            "$ref": "./examples/KustoClustersCreateOrUpdate.json"
+          }
+        },
+        "x-ms-long-running-operation": true,

```

Comment:
>x-ms-long-running-operation [](start = 9, length = 27)

So, we would like to provide async behavior with '201 + provisioningState'.. not 202. ?.. just confirming

Reply:
Correct